libs/community/langchain_community/vectorstores/infinispanvs.py

```diff
@@ -472,15 +573,15 @@ def put(self, key: str, data: str, cache_name: str) -> requests.Response:
             An http Response containing the result of the operation
         """
         api_url = self._default_node + self._cache_url + "/" + cache_name + "/" + key
-        response = requests.put(
+        response = self._h2c.put(
             api_url,
-            data,
+            content=data,
             headers={"Content-Type": "application/json"},
             timeout=REST_TIMEOUT,
         )
         return response
 
-    def get(self, key: str, cache_name: str) -> requests.Response:
+    def get(self, key: str, cache_name: str) -> Any:
```

Comment:
why the change to `Any` from `requests.Response`?

Reply:
when I started this PR the httpx lib was optional, so the idea was to allow both httpx.Response and request.Response. But now there's only httpx.Response.
Changed, thanks!